docs/infrastructure/infra-ui.asciidoc

```diff
@@ -2,67 +2,112 @@
 
 [float]
 [[infra-date]]
-=== Specify the date range
+=== Specify the time and date
 
-Use the time selector to focus on a specific timeframe.
+Click the time selector to choose the timeframe for the metrics.
+The values shown are the values for the last minute at the specified time and date.
 
 [float]
 [[infra-refresh]]
-=== Auto-refresh or pause
+=== Auto-refresh metrics
 
-Set auto-refresh to keep up-to-date information coming in, or stop
-refreshing to focus on historical data without new distractions.
+Select *Auto-refresh* to keep up-to-date metrics information coming in, or *Stop refreshing* to focus on historical data without new distractions.
 
 [float]
 [[infra-configure-source]]
-=== Adapt to your metric source
+=== Configure the data to use for your metrics
+
+The default source configuration for metrics is specified in the {kibana-ref}/infrastructure-ui-settings-kb.html[Infrastructure UI settings] in the {kibana-ref}/settings.html[Kibana configuration file].
+The default configuration uses the `metricbeat-*` index pattern to query the data.
+The default configuration also defines field settings for things like timestamps and container names.
+
+If your metrics have custom index patterns, or use non-default field settings, you can override the default settings.
+Click *Configuration* to change the settings.
+This opens the *Configure source* fly-out dialog.
+
+NOTE: These settings are shared with logs. Changes you make here may also affect the settings used in the *Logs* tab.
+
+In the *Configure source* dialog, you can change the following values:
+
+* *Name*: the name of the source configuration
+* *Indices*: the index pattern or patterns in the Elasticsearch indices to read metrics data and log data
+  from
+* *Fields*: the names of specific fields in the indices that need are used to query and interpret the data correctly
+
+TIP: If <<xpack-spaces>> are enabled in your Kibana instance, any configuration changes you make here are specific to the current space.
+You can make different subsets of data available by creating multiple spaces with different data source configurations.
+
+TIP: If you don't see the *Configuration* option, you may not have sufficient privileges to change the source configuration.
+For more information see <<xpack-security-authorization>>.
+
+[float]
+[[infra-metrics-explorer]]
+=== Visualize multiple metrics in Metrics Explorer
+
+<<metrics-explorer, Metrics Explorer>> allows you to visualize and analyze metrics for multiple components in a powerful and configurable way. Click the *Metrics Explorer* tab to get started.
+
+[float]
+[[infra-drill-down]]
+=== Drill down for related information
+
+Hover over a component to see more information about that component.
+
+Click a component to see the other actions available for that component.
+You can:
+
+* select *View Metrics* to <<xpack-view-metrics, view detailed infrastructure metrics for that component>>.
```

Comment:
Should these four bullets be capitalized? "Select"?

Reply:
Good spot!